app/assets/scripts/styles/inpage.js

```diff
@@ -45,8 +45,13 @@ export const InpageHeader = styled.header.attrs({
 
 export const InpageHeaderSticky = styled(InpageHeader)`
   position: sticky;
-  top: 0;
   z-index: 7000;
+
+  top: calc(40px + 0.75rem * 2);
```

Comment:
@danielfdsilva There might be a better way to calculate this using UI-library functions